### PR TITLE
data: AbhinavJaintle

### PR DESCRIPTION
**Pull request by AbhinavJaintle**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/7426"></a>

### CODE REVIEW

File: data/AbhinavJaintle [hunk 1]
```diff
@@ -0,0 +1,98 @@
+{
+  "name": "Abhinav Jain",
+  "type": "personal",
+  "displayStatsPublic": true,
```
Comment (khushalarora) [inline data/AbhinavJaintle:4]: You can remove these lines, these are not required now :+1: 
```Suggestion
```

File: data/AbhinavJaintle [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+  "bio": "Open Source Contributor **@CDLI** | Competitive Programmer | Public Speaker | Contest Developer  **@USACOGuide** | **Microsoft** LSA",
+  "avatar": "https://github.com/abhinavjaintle.png",
```
Comment (khushalarora) [inline data/AbhinavJaintle:6]: You can also remove this line, it is not required now :+1: 
```Suggestion
```

File: data/AbhinavJaintle [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+  "socials": [
+    { "icon": "FaTwitter", "url": "https://twitter.com/intent/user?screen_name=jaintle" },
+    { "icon": "FaGithub", "url": "https://github.com/abhinavjaintle" },
+    { "icon": "FaLinkedin", "url": "https://www.linkedin.com/in/jaintle/" },
+    { "icon": "FaYoutube", "url": "https://youtube.com/@abhinavjaincodes?sub_confirmation=1" }
+  ],
+  "links": [
+    {
+      "name": "Medium: Read my blogs",
+      "https://jaintle.medium.com/",
```
Comment (khushalarora) [inline data/AbhinavJaintle:24]: Add **`"url":`** in the beginning :+1: 
```Suggestion
     "url": "https://jaintle.medium.com/",
```

**Review by khushalarora — CHANGES_REQUESTED** [review]
Hey @AbhinavJaintle, thank you for contributing :tada: 
Please rename your file and Add **`.json`** at the end [Here](https://github.com/AbhinavJaintle/LinkFree/edit/AbhinavJaintle-patch-1/data/AbhinavJaintle).
Your filename will look like **`data/AbhinavJaintle.json`** :+1:

**Comment by AbhinavJaintle** [comment]
> Hey @AbhinavJaintle, thank you for contributing tada Please rename your file and Add **`.json`** at the end [Here](https://github.com/AbhinavJaintle/LinkFree/edit/AbhinavJaintle-patch-1/data/AbhinavJaintle). Your filename will look like **`data/AbhinavJaintle.json`** +1

Thanks for the help and suggestion folks @SaraJaoude @khushalarora :) 
I hope everything is in order now.

File: data/AbhinavJaintle.json [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+      "description": "Skills: Test Cases · Programming · OptimizationSkills: Test Cases · Programming · Optimization",
+      "url": "https://usaco.guide/"
+    },
+    {
+      "title": "**Microsoft** Learn Student Ambassadors",
+      "image": "https://github.com/abhinavjaintle.png",
+      "date": "Oct 2022",
+      "icon": "FaLink",
+      "color": "red",
+      "description": "",
```
Comment (kumarsonsoff3) [inline data/AbhinavJaintle.json:67]: Please add a brief description of the milestone here. Rest looks good to me!

Comment (AbhinavJaintle) [reply data/AbhinavJaintle.json:67]: Have added the description @kumarsonsoff3

**Review by SaraJaoude — APPROVED** [review]
looks good

**Pull request merged**